daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala

```diff
@@ -166,7 +166,7 @@ object ValueGenerators {
   } yield Bytes fromByteArray ab
 
   val cidV0Gen: Gen[ContractId.V0] =
-    Gen.alphaStr.map(t => Value.ContractId.V0.assertFromString('#' +: t))
+    Gen.alphaStr.map(t => Value.ContractId.V0.assertFromString('#' +: t.take(254)))
```

Comment:
What is this about? And why is it 254 here but 255 below? Does the 255 include the `#`?

Reply:
form the spec. 
```
  V0 Contract identifiers:
          cidV0  ∈  #[a-zA-Z0-9\._:-#/ ]{0,254}     -- V0ContractId
```

Reply:
I am a bit worried twe never reports failures for tests based on `ValueGenerators`. Before this changes  the  Generators `cidV0Gen` or `partyGen`  crash if the string generate `Gen.alphaStr` is too long.

Reply:
Maybe the value generators only generate short strings?

Reply:
It can generates string longer that 255, it is just unlikely.